lib/aptible/cli/subcommands/deploy.rb

```diff
@@ -29,6 +29,10 @@ def self.included(thor)
                    desc: 'Detach this app from its git repository: ' \
                          'its Procfile, Dockerfile, and .aptible.yml will be ' \
                          'ignored until you deploy again with git'
+            option :container_count, type: :numeric
+            option :container_size, type: :numeric
+            option :container_profile, type: :string,
+                                       desc: 'Supported types: m5 c5 r5'
```

Comment:
Request change: Indicate that these options only affect new services created by the deploy, not existing services, in their descriptions. As-is, it's not clear what the options do.